schemas/com.snowplowanalytics.snowplow.badrows/enrichment_failures/jsonschema/1-0-0

```diff
@@ -0,0 +1,311 @@
+                      "value": {
+                        "type": [ "string", "null" ],
+                        "description": "Stringified representation of the value which did not meet the enrichment's expectations"
+                      },
+                      "expectation": {
+                        "type": "string",
+                        "description": "Description of the expectation which was not met"
+                      }
+                    },
+                    "required": ["field", "value", "expectation" ],
+                    "additionalProperties": false
+                  },
+                  {
+                    "type": "object",
+                    "description": "Error which was internal to the enrichment regarding the enrichment's output data, this is a schema validation error",
+                    "properties": {
+                      "errors": {
+                        "type": "array",
+                        "minItems": 1,
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "path": {
+                              "type": "string",
+                              "description": "JSON Path to an issue in the faulty JSON Schema"
+                            },
+                            "message": {
+                              "type": "string",
+                              "description": "Human-readable message describing the issue with the schema"
+                            }
+                          },
+                          "required": [ "path", "message" ],
+                          "additionalProperties": false
+                        }
+                      }
+                    },
+                    "required": [ "errors" ],
+                    "additionalProperties": false
+                  },
+                  {
+                    "type": "object",
+                    "description": "Error which was external to the enrichment, e.g. a connection issue",
+                    "properties": {
+                      "error": {
+                        "type": "string",
+                        "description": "Error that occurred"
+                      }
+                    },
+                    "required": [ "error" ],
+                    "additionalProperties": false
+                  }
+                ]
+              }
+            },
+            "required": [ "enrichment", "message" ],
+            "additionalProperties": false
+          }
+        }
+      }
+    },
+    "payload": {
+      "type": "object",
+      "description": "The payload that resulted in enrichment failures",
+      "properties": {
+        "trackerProtocolPayload": {
+          "type": "object",
+          "description": "The tracker protocol payload that resulted in enrichment failures",
+          "properties": {
+            "app_id": { "type": ["string", "null"] },
```

Comment:
In case this is still supposed to be tracker protocol - what happens if new tracker (e.g. with implemented 1-0-5 [payload_data](https://github.com/snowplow/iglu-central/tree/master/schemas/com.snowplowanalytics.snowplow/payload_data/jsonschema) with new `foo` property) sends an event to a legacy enricher (that is aware only of 1-0-4). Will enricher just strip away this `foo` in a bad row?

Reply:
no enricher just won't care

Reply:
Not sure I following. It means we'll not have it them in a bad row?